fuse/requirements.txt

```diff
@@ -8,7 +8,7 @@ matplotlib>=3.3.3
 scikit-learn>=0.23.2
 termcolor>=1.1.0
 pycocotools>=2.0.1
-pytorch_lightning>=1.6
+pytorch_lightning>=1.6,<2.0.0 # temp - need to make all tests pass with this version first
```

Comment:
not related to this PR, but generally speaking PL version should be higher to fully support DDP..

Reply:
please add this request to #292